verifiers/internal/gha/builder.go

```diff
@@ -235,6 +233,32 @@ type WorkflowIdentity struct {
 	Issuer string
 }
 
+// SubjectWorkflowName returns the subject workflow without the git ref.
+func (id *WorkflowIdentity) SubjectWorkflowName() string {
+	withoutRef, err := url.Parse(id.SubjectWorkflow.String())
+	if err != nil {
+		// This should never happen.
+		panic(err)
```

Comment:
avoid panic'ing? Especially when if folks run our code as a service in the future. Wdut? Return "" or an error?

Reply:
I think panicking is often a good idea for things that should never ever happen. In this case an already parsed URL shouldn't ever be unparseable.

It seems that I should also be able to copy the URL safely so I will go ahead and do that instead.
https://github.com/golang/go/issues/38351

```golang
withoutRef := *id.SubjectWorkflow
```

Reply:
Done.